iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -282,13 +282,10 @@ public UserInstruction USER() {
     return b.<UserInstruction>nonterminal(DockerLexicalGrammar.USER).is(
       f.user(
         b.token(DockerKeyword.USER),
-        b.token(DockerLexicalGrammar.USER_NAME),
-        b.optional(
-          f.tuple(
-            b.token(DockerLexicalGrammar.USER_SEPARATOR),
-            b.token(DockerLexicalGrammar.USER_GROUP)
-          )
-        )
```

Comment:
Are these grammar rules still used? If not, let's drop them off to keep everything clean.